src/main/scala/org/constellation/util/CommonEndpoints.scala

```diff
@@ -73,15 +77,29 @@ trait CommonEndpoints extends Json4sSupport {
         )(complete(_))
       } ~
       path("info") {
-        val getInfo = dao.snapshotService.getSnapshotInfo.flatMap { info =>
-          info.acceptedCBSinceSnapshot.toList.traverse {
-            dao.checkpointService.fullData(_)
-          }.map(cbs => KryoSerializer.serializeAnyRef(info.copy(acceptedCBSinceSnapshotCache = cbs.flatten)).some)
-        }
+        APIDirective.extractHostPort(socketAddress) { hostPort =>
```

Comment:
This is an endpoint in `CommonEndpoints` which means it can be executed by both Peer and User.
If you're relying on the `idLookup` and observations, you should distinguish both executions and most likely move the endpoint from `CommonEndpoints` to `PeerAPI` and `API`